tests/runner.js

```diff
@@ -52,17 +52,17 @@ function runMocha() {
   mocha.run((failures) => {
     console.timeEnd('Mocha Tests Running Time');
 
-    // eslint-disable-next-line n/no-process-exit
-    process.exit(failures);
+    console.log(`[Mocha Debug]`, { failures });
+    // On Windows, process.exit at all causes exit code 1
+    if (failures > 0) {
+      // eslint-disable-next-line n/no-process-exit
+      process.exit(1);
+    }
   });
 }
 
-Promise.resolve()
-  .then(() => runMocha())
```

Comment:
original code on the left here doesn't make sense, because runMocha doesn't return a promise.